presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -601,14 +601,16 @@ public static long random(@SqlType(StandardTypes.BIGINT) long value)
         return ThreadLocalRandom.current().nextLong(value);
     }
 
-    @Description("a pseudo-random number from a normal distribution with mean, std, and p")
-    @ScalarFunction(alias = "rand_normal", deterministic = false)
+    @Description("Compute the inverse of the Normal cdf with given mean and\n" +
```

Comment:
This should be a concise single-line description. See other examples in the same file. This is what will show up in the output of SHOW FUNCTIONS.